Automattic/jetpack
# ThreatsDataViews: Improve responsiveness

## Description
Ensure the `ThreatsDataViews` component appropriately handles various view ports.

### Other information:

## Jetpack product discussion


- https://github.com/Automattic/jetpack-scan-team/issues/1597

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






- Review `ThreatsDataViews` storybooks and ensure no regressions are introduced
- Checkout this Protect branch on Jurassic Tube/Ninja
- View the `ScanReport` and `ScanResultsDataViews` in varying screen sizes and ensure the search, filter and toggle actions and the table all respond correctly and the content is appropriately spaced out, or scrollable
- When viewing the `ScanResultsDataViews` component in mobile (`max-width: 599px`), ensure that the component defaults to list view.

## Screenshots

ScanResultsDataViews:
![Screen Shot on 2024-12-18 at 10-48-16](https://github.com/user-attachments/assets/c0fe76a2-55f8-47ba-b680-c527803fd15c)

ScanReport:
![Screen Shot on 2024-12-18 at 10-48-48 (1)](https://github.com/user-attachments/assets/b54f2a85-7bcb-499b-b394-c77fb50f6387)
